### ISSUE TITLE
Template images in tray not displayed at full opacity

### ISSUE THREAD

**Issue by tommoor** [electron/electron#2387]
It is no longer possible to change the tray icon whilst the app is running, this is true in version 0.30.1 & 0.30.2. This should be reproducible on OSX as simply as:

```
var tray = new Tray("../path/to/img");
tray.setImage("../path/to/new-image");
```

Everything works as expected in Electron v0.30.0.
Related to https://github.com/atom/electron/issues/2360

**Comment by hokein** [comment]
@tommoor 
I can't reproduce it in v0.30.2. The above code works and I can see the tray icon has changed.

**Comment by tommoor** [comment]
@hokein there must be another factor, I will try and reduce down our app until it starts working - something definitely regressed between versions though.

**Comment by tommoor** [comment]
After some more investigation it turns out that setImage is working, however Template images are not being displayed at the correct opacity (our two images were the same thing at different opacities).

**Comment by hokein** [comment]
@tommoor 
Still not reproduce it on OS X 10.10.4.

I use the two templates images you send me:

test2Template.png:

![test2template](https://cloud.githubusercontent.com/assets/2557445/9029728/2acb04c4-39cb-11e5-936a-6f421c6ec480.png)

test3Template.png:

![test3template](https://cloud.githubusercontent.com/assets/2557445/9029730/2d2eec44-39cb-11e5-8e46-73d30cbfa27e.png)
 
The test code is below:

```
var app = require('app');
var Tray = require('tray');
var path = require('path');
var BrowserWindow = require('browser-window');

var appIcon = null;
var win = null;

app.on('ready', function(){
  win = new BrowserWindow({show: false});
  //  test3Template.png or test2Template.png
  appIcon = new Tray(path.join(__dirname, 'test3Template.png'));
});
```

Here is the result:

test2Template.png with dark mode:

<img width="217" alt="qq20150803-3 2x" src="https://cloud.githubusercontent.com/assets/2557445/9029758/8332ee06-39cb-11e5-95cd-5c451f30ab14.png">

test2Template.png with light mode:
<img width="165" alt="qq20150803-4 2x" src="https://cloud.githubusercontent.com/assets/2557445/9029759/85bcaf9a-39cb-11e5-8bac-a31a0bc07ef4.png">

test3Template.png with dark mode:
<img width="165" alt="qq20150803-5 2x" src="https://cloud.githubusercontent.com/assets/2557445/9029761/88bbf34a-39cb-11e5-8f40-0b14bcf0a95b.png">

test3Template.png with light mode:
<img width="152" alt="qq20150803-6 2x" src="https://cloud.githubusercontent.com/assets/2557445/9029762/8d1f9090-39cb-11e5-8003-59c723fbfebe.png">

**Comment by tommoor** [comment]
I still don't know what the difference is, but as it can't be independently reproduced I'll close this issue for now - If I get a good working test I'll reopen. Thanks for your time @hokein

**Comment by tommoor** [comment]
I'm still repro'ing this in 0.30.4 with the exact code and images from above, so are the other members of my team. Anyone else able to give this a try?

**Comment by tommoor** [comment]
@hokein which version of OSX are you testing on? The ones on our team that can repro this are on 10.10.1

**Comment by hokein** [comment]
I test on OS X 10.10.4.

**Comment by zcbenz** [comment]
I tested on 10.10.5 and changing the images works for me.

**Comment by paulatbipsync** [comment]
We are also experiencing this on 10.9, on 10.10 it works as expected
As a workaround we are not using 'Template' images on pre 10.10 osx

**Comment by zcbenz** [comment]
Closed with #4924.
